### PR TITLE
refactor: cli queries

### PR DESCRIPTION
**Pull request by jmastr**

### CODE REVIEW

File: proto/planetmintgo/dao/query.proto [hunk 1]
```diff
@@ -26,19 +26,19 @@ service Query {
...
   rpc GetMintRequestsByHash (QueryGetMintRequestsByHashRequest) returns (QueryGetMintRequestsByHashResponse) {
-    option (google.api.http).get = "/planetmint/dao/get_mint_requests_by_hash/{hash}";
+    option (google.api.http).get = "/planetmint/dao/mint_requests_hash/{hash}";
   
   }
   
   // Queries a list of MintRequestsByAddress items.
   rpc MintRequestsByAddress (QueryMintRequestsByAddressRequest) returns (QueryMintRequestsByAddressResponse) {
-    option (google.api.http).get = "/planetmint/dao/mint_requests_by_address/{address}";
+    option (google.api.http).get = "/planetmint/dao/mint_requests_address/{address}";
```
Comment (eckelj) [inline proto/planetmintgo/dao/query.proto:35]: shouldn't we name this one to
planetmint/dao/ mint_requests/address/{address} like the others below?

File: proto/planetmintgo/machine/query.proto [hunk 2]
```diff
@@ -22,25 +22,25 @@ service Query {
...
   rpc GetMachineByPublicKey (QueryGetMachineByPublicKeyRequest) returns (QueryGetMachineByPublicKeyResponse) {
-    option (google.api.http).get = "/planetmint/machine/get_machine_by_public_key/{publicKey}";
+    option (google.api.http).get = "/planetmint/machine/public_key/{publicKey}";
   
   }
   
   // Queries a list of GetTrustAnchorStatus items.
   rpc GetTrustAnchorStatus (QueryGetTrustAnchorStatusRequest) returns (QueryGetTrustAnchorStatusResponse) {
-    option (google.api.http).get = "/planetmint/machine/get_trust_anchor_status/{machineid}";
+    option (google.api.http).get = "/planetmint/machine/trust_anchor_status/{machineid}";
```
Comment (eckelj) [inline proto/planetmintgo/machine/query.proto:31]: shouldn't we use

"/planetmint/machine/trust_anchor/status/{machineid}";

**Review by eckelj — CHANGES_REQUESTED** [review]
@jmastr I totally like the improvement! I just have two comments to be improved and thus aligns with the other improvements.

File: proto/planetmintgo/dao/query.proto (hunk 1 above)
Comment (jmastr) [reply proto/planetmintgo/dao/query.proto:35]: you're completely right. however the corresponding cli command needs then turned from `planetmint-god query dao mint-requests-address` into `planetmint-god query dao mint-requests address`. Meaning `address` needs to be a sub-commmand of mint-requests....

I'll have a look how to do that...

**Comment by jmastr** [comment]
> @jmastr I totally like the improvement! I just have two comments to be improved and thus aligns with the other improvements.

@eckelj done

**Review by eckelj — APPROVED** [review]

**Pull request merged**